app/models/service.rb

```diff
@@ -121,12 +121,13 @@ def vms
 
   def start
     raise_request_start_event
-    queue_group_action(:start)
+    queue_group_action(:start, 0, 1, delay_for_action(0, :start))
   end
 
   def stop
     raise_request_stop_event
-    queue_group_action(:stop, last_group_index, -1)
+    last_index = last_group_index
+    queue_group_action(:stop, last_index, -1, delay_for_action(last_index, :stop))
```

Comment:
Should this apply to the `suspend` and `shutdown_guest` methods as well?  If they do I think then the default parameters for `queue_group_action` can be removed since they are always being passed.